addons/knobs/src/components/Panel.tsx

```diff
@@ -220,6 +220,7 @@ export default class KnobPanel extends PureComponent<KnobPanelProps> {
     const groupIds: string[] = [];
 
     const knobKeysArray = Object.keys(knobs).filter(key => knobs[key].used);
+    const knobsArray = knobKeysArray.map(key => knobs[key]);
```

Comment:
This was actually wrong 😮

Reply:
@ndelangen this needs its own PR

Reply:
ESlint will complain then. I'd have to add a eslint ignore line in this PR, then merge this and then create a new PR that removes the ignore line and moves the code a few lines up?

Reply:
It just feels more efficient to resolve the ESlint warning immediately instead of ignoring it and then addressing it immediately afterwards.

Reply:
I think he's saying fix the "wrong" issues in a separate PR, and then merge master into this PR and then it'd just be ESLint fixes and ESLint config integration

Reply:
ok